### DISCUSSION TITLE
Request for function in outline view.

### DISCUSSION THREAD

**Discussion by Lledor** [vkbo/novelWriter#2093] [Ideas]
Hi, I just wanted to express how wonderful this application is.

Would it be possible to add a word wrap option in the outline view, specifically in the synopsis section? It would really enhance the clarity of structural planning.

Thank you so much for making this app open-source! I believe it will only continue to gain popularity as more people switch to Linux distributions. So far, this program is the only truly viable option I've found.

**Comment by vkbo** [comment]
There is already a feature request for that in #1465, but the tree view used for generating the outline view does not have that capability, so it can't be done with the current Outline. I would have to create a new implementation to change this behaviour, and I am considering it. There are also requests for more visual representations of the story structure, which may anyway be a better solution to this.

**Reply by Lledor** [reply]
A clearer, more visual way to organize thoughts within the program would be incredibly helpful for writers like me. With dyscalculia, I often lean on my visual-spatial skills rather than linear methods, so a flexible, visual structure would make organizing ideas feel more natural. Thank you for considering this support for diverse thinking styles.
